### PR TITLE
Add Organizational Data link to myteam-explore.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/issues/273.

### CODE REVIEW

File: Viva/insights/Use/myteam-explore.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ audience: Admin
...
 
-The **Group insights** section is available to managers who the Viva Insights admin specifies. Group insights are based all employees that report up to you as the manager, not just your immediate group members or direct reports. The admin uses the organizational data that is uploaded into Viva Insights in Workplace Analytics or that’s presented in Azure AD.
+The **Group insights** section is available to managers who the Viva Insights admin specifies. Group insights are based all employees that report up to you as the manager, not just your immediate group members or direct reports. The admin uses the organizational data that is uploaded into Viva Insights in Workplace Analytics or that’s presented in Azure AD. For more information, see [Organizational data](organizational-data.md).
```
Comment (JohanFreelancer9) [inline Viva/insights/Use/myteam-explore.md:22]:
```suggestion
The **Group insights** section is available to managers whom the Viva Insights admin specifies. Group insights are based on all employees that report up to you as the manager, not just your immediate group members or direct reports. The admin uses the organizational data that is uploaded into Viva Insights in Workplace Analytics or that’s presented in Azure AD. For more information, see [Organizational data](organizational-data.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - not related, but additional changes, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@madehmer PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
Closing because this article has been deleted. 
Redirect points to: 
https://github.com/MicrosoftDocs/Viva/blob/public/Viva/insights/org-team-insights/team-insights.md.

**Pull request closed without merging**